Cargo.lock

```diff
@@ -2015,9 +2039,15 @@ checksum = "1dd624098567895118886609431a7c3b8f516e41d30e0643f03d94592a147e36"
 
 [[package]]
 name = "unicode-width"
-version = "0.1.10"
+version = "0.1.11"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "c0edd1e5b14653f783770bce4a4dabb4a5108a5370a5f5d8cfe8710c361f6c8b"
+checksum = "e51733f11c9c4f72aa0c160008246859e340b00807569a0da0e7a1079b27ba85"
+
+[[package]]
+name = "unsafe-libyaml"
+version = "0.2.9"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "f28467d3e1d3c6586d8f25fa243f544f5800fec42d97032474e17222c2b75cfa"
```

Comment:
Actually, could you roll back this part of the update? We'll drop yml anyway, and I don't want to deal with this mess. It's a libyaml transpiled into rust, so basically a 'nice way to add a security nightmare' sort of things.

Reply:
I mean yaml is just a security nightmare anyway. I don't see how keeping an unmaintained version would be more secure.

And yaml probably isn't going away soon.

Reply:
I mean, they are both not maintained. It's just one is a massive transpilation of a C library, and the rust crate is a small safe rust parser. They're not maintained from the same year.